frontend/src/pages/modelCatalog/ModelCatalogCoreLoader.tsx

```diff
@@ -2,60 +2,45 @@ import * as React from 'react';
 import { Outlet } from 'react-router';
 import { conditionalArea, SupportedArea } from '~/concepts/areas';
 import ApplicationsPage from '~/pages/ApplicationsPage';
-import TitleWithIcon from '~/concepts/design/TitleWithIcon';
 import { ProjectObjectType, typedEmptyImage } from '~/concepts/design/utils';
 import {
   ModelCatalogContext,
   ModelCatalogContextProvider,
 } from '~/concepts/modelCatalog/context/ModelCatalogContext';
+import TitleWithIcon from '~/concepts/design/TitleWithIcon';
 import EmptyModelCatalogState from './EmptyModelCatalogState';
 
-type ApplicationPageProps = React.ComponentProps<typeof ApplicationsPage>;
-
-type ApplicationPageRenderState = Pick<
-  ApplicationPageProps,
-  'emptyStatePage' | 'empty' | 'headerContent'
->;
 const ModelCatalogCoreLoader: React.FC = conditionalArea(
   SupportedArea.MODEL_CATALOG,
   true,
 )(() => {
   const { modelCatalogSources } = React.useContext(ModelCatalogContext);
-  let renderStateProps: ApplicationPageRenderState & { children?: React.ReactNode };
+
   if (modelCatalogSources.length === 0) {
-    renderStateProps = {
-      empty: true,
-      emptyStatePage: (
-        <>
-          <EmptyModelCatalogState
-            testid="empty-model-catalog-state"
-            title="Request access to model catalog"
-            description="To request access to model catalog, contact your administrator."
-            headerIcon={() => (
-              <img src={typedEmptyImage(ProjectObjectType.registeredModels)} alt="" />
-            )}
-          />
-        </>
-      ),
-      headerContent: null,
-    };
-  } else {
     return (
-      <ModelCatalogContextProvider>
-        <Outlet />
-      </ModelCatalogContextProvider>
+      <ApplicationsPage
+        title={
+          <TitleWithIcon title="Model Catalog" objectType={ProjectObjectType.registeredModels} />
+        }
+        loaded
+        empty
+        provideChildrenPadding
+      >
+        <EmptyModelCatalogState
+          testid="empty-model-catalog-state"
+          title="Request access to model catalog"
+          description="To request access to model catalog, contact your administrator."
+          headerIcon={() => (
+            <img src={typedEmptyImage(ProjectObjectType.registeredModels)} alt="" />
+          )}
+        />
+      </ApplicationsPage>
     );
   }
```

Comment:
@YuliaKrimerman this change will conflict with your PR https://github.com/opendatahub-io/odh-dashboard/pull/3743, but no matter which order these merge the result should be as it is in 3743 - this component renders only
```
      <ModelCatalogContextProvider>
        <Outlet />
      </ModelCatalogContextProvider>
```
and we'll rely on the two page components to handle the empty state (which they already do).

Reply:
@mturley you want me to update this here in this PR? or is this something will handle it in 3743?

Reply:
Sorry I just saw this - it doesn't particularly matter as long as we end up there when both are merged. I think we can merge this as-is since it's already approved and then @YuliaKrimerman will have to address the merge conflict...... sorry @YuliaKrimerman !!